### PR TITLE
I added six commit.

### PR DESCRIPTION
**Pull request by dbush001**

### CODE REVIEW

**Comment by mbMosman** [comment]
You just need to use one of the emoji from this "cheat-sheet" just don't put the :'s on each side of it.  So like your file submitted here, but with a emoji from the list.  There is a older_man...  So you could do:

`emoji: older_man` and it would show up.

File: _data/fall_2016/dbush001.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: David Bush
+emoji: older man
```
Comment (mbMosman) [inline _data/fall_2016/dbush001.yml]: There is an `older_man`.  You just needed the underscore if that's what you were aiming for.

**Comment by dbush001** [comment]
Hello

I like to know if there is away to delete a html file from GitHub. I push my fan-page to GitHub and I forgot to validation it so I had to go back and fix my page and push it again. But know when I do a pull request it pulls both files.

________________________________

**Comment by Phaithao** [comment]
Go to GitHub and go to the file you're working on, make sure it says your
name on the left side/html students. Click on options then scroll all the
way down, you should see an option to delete it there.

**Comment by mbMosman** [comment]
Yep, that would work to delete from GitHub, but then you'll need to do a `git pull` on your PC to get that change there.  What I would suggest instead is to delete that file on your computer.  That delete will be added when you do the add/commit/push.  `git add .` adds new files, changed files, and deleted files.

**Pull request merged**